src/index.html

```diff
@@ -11,6 +11,50 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="search__bar big__bar"
+      method="post"
+      data-qa="big"
+    >
+      <label
+        for="biginput"
+        class="big"
+        >
+        <img
+        src="./images/Search.svg"
+        alt="search"
+        class="big__icon"
+        >
+        <input
+          class="search__input big--input"
+          type="text"
+          name="biginput"
+          data-qa="keypress"
+          placeholder="Try “Los Angeles”"
+        >
+      </label>
```

Comment:
```suggestion
      <label for="biginput" class="big">
        <img
          src="./images/Search.svg"
          alt="search"
          class="big__icon"
        >
        
        <input
          class="search__input big--input"
          type="text"
          name="biginput"
          data-qa="keypress"
          placeholder="Try “Los Angeles”"
        >
      </label>
```

Reply:
Check all other similar cases. Fix formatting

Reply:
not fixed